From 0d72150d0f1e6d52c3b8888500fbcce64c751362 Mon Sep 17 00:00:00 2001
From: "Lucas A. M. Magalhaes" <lamm@linux.ibm.com>
Date: Fri, 21 Feb 2020 09:40:28 -0300
Subject: [PATCH] test: Fix obj_zones for ppc64le

This test was timing out because of intensive page_fault
handling. By allocating more by each loop we minimize this
issue.

Signed-off-by: Lucas A. M. Magalhaes <lamm@linux.ibm.com>
---
 src/test/obj_zones/obj_zones.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/test/obj_zones/obj_zones.c b/src/test/obj_zones/obj_zones.c
index c1d84268a..3cc796beb 100644
--- a/src/test/obj_zones/obj_zones.c
+++ b/src/test/obj_zones/obj_zones.c
@@ -36,6 +36,7 @@
  */
 
 #include <stddef.h>
+#include <page_size.h>
 
 #include "unittest.h"
 
@@ -104,7 +105,7 @@ test_malloc_free(const char *path)
 			0, S_IWUSR | S_IRUSR)) == NULL)
 		UT_FATAL("!pmemobj_create: %s", path);
 
-	size_t alloc_size = 128 * 1024;
+	size_t alloc_size = PMEM_PAGESIZE * 32;
 	size_t max_allocs = 1000000;
 	PMEMoid *oid = MALLOC(sizeof(PMEMoid) * max_allocs);
 	size_t n = 0;
-- 
2.20.1

